test/commonControlTests/NumberControl.spec.ts

```diff
@@ -11,608 +11,436 @@
-        });
+    test('Input value fail validation check', async () => {
+        const requestHandler = new ControlHandler(new AgeControlManager());
+        await testE2E(requestHandler, [
+            'U: set my age to five',
+            TestInput.of(
+                ValueControlIntent.of(AmazonBuiltInSlotType.NUMBER, {
+                    'AMAZON.NUMBER': '5',
+                    action: $.Action.Set,
+                }),
+            ),
+            'A: Was that 5?',
+            'U: yes',
+            TestInput.of(IntentBuilder.of(AmazonIntent.YesIntent)),
+            "A: Sorry but that's not a valid choice because the age must be even. How old will you be?",
+            'U: fourteen',
+            TestInput.of(
+                ValueControlIntent.of(AmazonBuiltInSlotType.NUMBER, {
+                    'AMAZON.NUMBER': '14',
+                }),
+            ),
+            'A: Great.',
+        ]);
+        expect(requestHandler.getSerializableControlStates().ageSelector.value).eq(14);
+    });
 
-        test('first number strictly disaffirmed, propose user with ambiguous pair based on the first number', async () => {
-            const requestHandler = new ControlHandler(new NumberControlManager());
-            await testE2E(requestHandler, [
-                'U: Fifteen',
-                TestInput.of(
-                    ValueControlIntent.of(AmazonBuiltInSlotType.NUMBER, {
-                        action: $.Action.Set,
-                        'AMAZON.NUMBER': '15',
-                    }),
-                ),
-                'A: Was that 15?',
-                'U: No, fifteen',
-                TestInput.of(
-                    ValueControlIntent.of(AmazonBuiltInSlotType.NUMBER, {
-                        feedback: $.Feedback.Disaffirm,
-                        'AMAZON.NUMBER': '15',
-                    }),
-                ),
-                'A: Sorry you disagreed but with the same number. Did you perhaps mean 50?',
-                'U: Yes',
-                TestInput.of(IntentBuilder.of(AmazonIntent.YesIntent)),
-                'A: Great.',
-            ]);
-            expect(
-                (requestHandler.getSerializableControlStates()[TEST_CONTROL_ID] as NumberControlState).value,
-            ).eq(50);
-        });
+    test('Clear the value on Number Control', async () => {
+        const requestHandler = new ControlHandler(new AgeControlManager());
+        await testE2E(requestHandler, [
+            'U: set my age to four',
+            TestInput.of(
+                ValueControlIntent.of(AmazonBuiltInSlotType.NUMBER, {
+                    'AMAZON.NUMBER': '4',
+                    action: $.Action.Set,
+                }),
+            ),
+            'A: Was that 4?',
+            'U: yes',
+            TestInput.of(IntentBuilder.of(AmazonIntent.YesIntent)),
+            'A: Great.',
+            'U: Clear my age',
+            TestInput.of(GeneralControlIntent.of({ action: $.Action.Remove })),
```

Comment:
nit: "U: Clear " shouldn't produce Action.Remove.  please make the test title, utterance and action consistent.

Reply:
Fixed to use $.Action.Clear